bundle/manifests/cert-manager-operator.clusterserviceversion.yaml

```diff
@@ -742,13 +742,13 @@ spec:
   provider:
     name: Red Hat
   relatedImages:
-  - image: quay.io/jetstack/cert-manager-webhook:v1.12.4
+  - image: quay.io/jetstack/cert-manager-webhook:v1.13.1
     name: cert-manager-webhook
-  - image: quay.io/jetstack/cert-manager-cainjector:v1.12.4
+  - image: quay.io/jetstack/cert-manager-cainjector:v1.13.1
     name: cert-manager-ca-injector
-  - image: quay.io/jetstack/cert-manager-controller:v1.12.4
+  - image: quay.io/jetstack/cert-manager-controller:v1.13.1
     name: cert-manager-controller
-  - image: quay.io/jetstack/cert-manager-acmesolver:v1.12.4
+  - image: quay.io/jetstack/cert-manager-acmesolver:v1.13.1
     name: cert-manager-acmesolver
-  replaces: cert-manager-operator.v1.11.4
-  version: 1.12.0
+  replaces: cert-manager-operator.v1.12.0
```

Comment:
TODO: replaces field should be changed to use v1.12.1, as now we expect v1.12.1 to be released before v1.13.0.

Reply:
And `    olm.skipRange: '>=1.12.0 <1.13.0'` should be changed to use v1.12.1 too?

Reply:
yes, should be '>=1.12.1 <1.13.0'

Reply:
Updated in latest.